cluster/ubuntu/util.sh

```diff
@@ -447,12 +447,45 @@ function kube-down {
   done
   wait
 }
-
 # Update a kubernetes cluster with latest source
 function kube-push {
-  echo "not implemented"
-}
 
+  #stop all the kube's process & etcd 
+  source "${KUBE_ROOT}/cluster/ubuntu/${KUBE_CONFIG_FILE-"config-default.sh"}"
+
+  for i in ${nodes}; do
+    echo "Cleaning on node ${i#*@}"
+    ssh -t $i 'sudo -p "[sudo] stop all process: " service etcd stop' || true
+    ssh -t $i 'rm -f /opt/bin/kube* /etc/init/kube* /etc/init.d/kube* /etc/default/kube*; rm -rf ~/kube' || true
+  done
+
+  # Run build.sh to get the latest release
+  source "${KUBE_ROOT}/cluster/ubuntu/build.sh"
```

Comment:
I guess it's okay for now, but this should be implemented in a follow up PR and we should make a note in this PR that it doesn't follow the kube-push usage.